_posts/digests/2019-09-22-32.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Last Week in AI #32"
+excerpt: "Deepfake anonymization, measuring the carbon impact of AI, and more!"
+image:
+  feature: assets/img/digests/32/deepfake-anon.jpg
+  credit: <a href="<Image Source Link>"> <Author> / <Source Name>
+categories: [digest]

```

Comment:
should be "digests" - could you pull the latest csv parser repo it's been fixed there